CONTRIBUTING.md

```diff
@@ -15,18 +15,23 @@ cd dnsimple-python
 
 Make sure you have Python (7.3 onwards) installed.
```

Comment:
I think this version number is a typo

Reply:
You never heard of the new super-python 7.3? 🤣 

ooooooops that's actually a PHP version 🤦

Reply:
Fixed